### ISSUE TITLE
(Feature Request): Dislike reported by the creator

### ISSUE THREAD

**Issue by EricZhang456** [Anarios/return-youtube-dislike#531]
### Extension or Userscript?

Both

### Request or suggest a new feature!

![image](https://user-images.githubusercontent.com/32814750/163509691-5c914071-092f-4ac4-9e51-4943d327bab9.png)

As shown (dislike count highlighted), creators can see the dislike count, which can probably solve the problems on new videos.

### Ways to implement this!

Have creators grant access, read the dislike, store them somewhere and display it to the viewer

### Can you work on this?

- [ ] Yes
- [X] No

**Comment by sy-b** [comment]
Similar issues: #56, #84, #392

Related issue: #130,

**Comment by sy-b** [comment]
Verifying the reported dislikes is hard

This requires at least 3 prerequisites -
1. Verifying that the reported vote count isn't being tampered with
2. Creator's privacy  i.e. assuring them that nothing other than what is needed is being collected
3. Quick & frictionless setup for creators

Creating a solution with any 2 of these at a time is easily possible but implementing all 3 is a difficult thing.
Of course, there have been discussions & we have a solution that can ensure all the 3 points which _may_ work but we are not so sure. And also there is no proof of concept for the proposed solution(s). Also, the build effort required _might be_ quite high (not sure about this).

As of now, the dislike extrapolation is pretty good for most of the videos with a good amount of audience. It is mostly 90% accurate. This number will increase with more people using & contributing their votes to the API. 

I think if creators want the dislike count to be more accurate for their videos then they should ask their viewers to use the extension (or any app **that submits votes** to this API).

**Comment by cyrildtm** [comment]
> I think if creators want the dislike count to be more accurate for their videos then they should ask their viewers to use the extension (or any app **that submits votes** to this API).

That's a good one. 

Implementing a secure, mutually trustable and convenient service that used to be available for free but is now gone because of corporate greed. The goal is noble but it sounds very discouraging. Why donate so much of my time just to restore what is ruined with a finger snap

**Comment by sy-b** [comment]
![LessWrong (RYD) — 01 12 2022 UTC  you want more accurate data - tell your users about the extension, and data will become more accurate](https://user-images.githubusercontent.com/94835959/163599819-cdbae876-c84d-41c7-ba82-2381e95db676.PNG)

**Comment by cyrildtm** [comment]
> A comment from discord discussion by the owner
> 
> ![LessWrong (RYD) — 01 12 2022 UTC you want more accurate data - tell your users about the extension, and data will become more accurate](https://user-images.githubusercontent.com/94835959/163599819-cdbae876-c84d-41c7-ba82-2381e95db676.PNG)

ever considered putting this to home page? maybe rephrase a bit, like "viewers" instead of "users"

**Comment by sy-b** [comment]
> ever considered putting this to home page? maybe rephrase a bit, like "viewers" instead of "users"

Good Idea.
